app/commands/donations/stripe/sync_subscriptions.rb

```diff
@@ -0,0 +1,48 @@
+# Ensure that our local Stripe subscriptions match the actual Stripe subscriptions
+class Donations::Stripe::SyncSubscriptions
+  include Mandate
+
+  def call
+    create_subscriptions!
+    deactivate_subscriptions!
+  end
+
+  def create_subscriptions!
+    missing_active_subscription_ids = active_stripe_subscription_ids - local_subscription_ids
+    missing_active_subscription_ids.each do |subscription_id|
+      subscription = active_stripe_subscriptions[subscription_id]
+
+      user = User.find_by(stripe_customer_id: subscription.customer)
+      next unless user
+
+      Donations::Subscription::Create.(user, subscription)
+    end
+  end
+
+  def deactivate_subscriptions!
+    inactive_subscription_ids = local_subscription_ids - active_stripe_subscription_ids
+    inactive_subscription_ids.each do |subscription_id|
+      subscription = local_subscriptions[subscription_id]
+      Donations::Subscription::Deactivate.(subscription) if subscription.active
```

Comment:
We need to figure out which statuses we'll deactivate for. The available values are:
`incomplete`, `incomplete_expired`, `trialing`, `active`, `past_due`, `canceled`, or `unpaid`.